Makefile

```diff
@@ -1,6 +1,52 @@
+help:
+	clear
+	@echo "Help"
+	@echo "make install - install istio"
+	@echo "make start   - install bookinfo"
+	@echo "make status  - show status"
+	@echo "make stop    - stop bookinfo"
+	@echo ""
+	@echo "Fault Injection:"
+	@echo "make tratings-test-delay - Injecting an HTTP delay fault"
+	@echo "make ratings-test-abort  - Injecting an HTTP abort fault"
 start:
+	clear
 	@echo "Start..."
+	wget https://raw.githubusercontent.com/istio/istio/release-1.1/samples/bookinfo/platform/kube/bookinfo.yaml
+	istioctl kube-inject -f bookinfo.yaml > bookinfo-istio.yaml
+	kubectl apply -f bookinfo-istio.yaml
+	wget https://raw.githubusercontent.com/istio/istio/release-1.1/samples/bookinfo/networking/bookinfo-gateway.yaml
+	kubectl apply -f bookinfo-gateway.yaml
+	wget https://raw.githubusercontent.com/istio/istio/release-1.1/samples/bookinfo/networking/destination-rule-all-mtls.yaml
+	kubectl apply -f destination-rule-all-mtls.yaml
+	wget https://raw.githubusercontent.com/istio/istio/release-1.1/samples/bookinfo/networking/virtual-service-all-v1.yaml
+	kubectl apply -f virtual-service-all-v1.yaml
+	weg https://raw.githubusercontent.com/istio/istio/release-1.1/samples/bookinfo/networking/virtual-service-reviews-test-v2.yaml
```

Comment:
I would suggest moving this to the separate task as it will override `virtual-service-all-v1.yaml`